tests/test_message_segmenting.py

```diff
@@ -52,22 +52,22 @@ def test_message_profiling_gsm7(self, short_gsm7_text, long_gsm7_text, byte_stri
     def test_message_profiling_ucs2(self, short_ucs2_text, long_ucs2_text):
         profiled_message = profiling.profile_message(short_ucs2_text)
         assert profiled_message["num_segments"] == 1
-        assert profiled_message["message_length"] == 14
```

Comment:
These are all changes going back to the way it originally was before it was [rewritten here](https://github.com/chrisconlon-klaviyo/sms-toolkit/commit/5e80ba60e9d5ea4ed8c379fce8e533d708e0a1bf#diff-5c1e15f9a12bc24f389e473c94186c89)